api/android/api/jni/Android-nnstreamer.mk

```diff
@@ -23,6 +23,9 @@ LOCAL_SRC_FILES := \
     $(NNSTREAMER_DECODER_IL_SRCS) \
     $(NNSTREAMER_DECODER_PE_SRCS)
 
+# Remove duplicates
+LOCAL_SRC_FILES := $(sort $(NNSTREAMER_SRC_FILES))
```

Comment:
should `sort -u` be used for unique list (to remove duplicates)?

Reply:
sort -u shows warning in ndk build (no option '-u')
https://www.gnu.org/software/make/manual/html_node/Text-Functions.html

Reply:
Yes, I was looking at GNU core-utils definition of make
https://www.gnu.org/software/coreutils/manual/html_node/sort-invocation.html
You are right.